utils/python/CIME/case.py

```diff
@@ -0,0 +1,80 @@
+"""
+Wrapper around all env XML for a case.
+
+All interaction with and between the module files in XML/ takes place
+through the Case module.
+"""
+
+from XML.standard_module_setup import *
+import CIME.utils
+from CIME.utils import expect, run_cmd
+from CIME.XML.machines import Machines
+
+from CIME.XML.env_test          import EnvTest
+from CIME.XML.env_mach_specific import EnvMachSpecific
+from CIME.XML.env_case          import EnvCase
+from CIME.XML.env_mach_pes      import EnvMachPes
+from CIME.XML.env_build         import EnvBuild
+from CIME.XML.env_run           import EnvRun
+from CIME.XML.env_archive       import EnvArchive
+from CIME.XML.env_batch         import EnvBatch
+
+class Case(object):
+
+    def __init__(self, case_root=os.getcwd()):
```

Comment:
So we are not accounting for creating a new case yet and this __init__ will in the future allow for case_root to not exist, right?